packages/angular/projects/angular/src/components/radio-button/radio-button.component.scss

```diff
@@ -0,0 +1,10 @@
+@import '@devextreme/styles/src/radio-button/radio-button.scss';
+
+:host {
+  display: block;
+}
+
+.radio-input {
```

Comment:
Question: Maybe we should move this to the style package?

Reply:
Yes, agree with you.
Fixed.